src/tag_index.c

```diff
@@ -226,12 +226,12 @@ static void TagReader_OnReopen(void *privdata) {
       InvertedIndex *idx = TagIndex_OpenIndex(ctx->idx, ir->record->term.term->str,
                                               ir->record->term.term->len, DONT_CREATE_INDEX, &sz);
       if (idx == TRIEMAP_NOTFOUND || ir->idx != idx) {
-        // the inverted index was collected entirely by GC, lets stop searching.
+        // the inverted index was collected entirely by GC.
         // notice, it might be that a new inverted index was created, we will not
```

Comment:
```suggestion
        // The inverted index was collected entirely by GC.
        // Notice, it might be that a new inverted index was created if all of the documents were 
        // removed and new ones were added.
        // We will not continue reading those old results and we do not promise that documents
        // that were added during cursor life will be returned by the cursor, although that can very well happen
```